build.gradle

```diff
@@ -46,9 +46,9 @@ dependencies {
         compile 'com.epam.reportportal:commons-rules:5.0.0-BETA-6'
         compile 'com.epam.reportportal:commons-model:5.0.0-BETA-8'
     } else {
-        compile 'com.github.reportportal:commons-dao:5.0.0-BETA-7'
+        compile 'com.github.reportportal:commons-dao:9b640a28'
```

Comment:
if there were no changes, please return back the release version

Reply:
done